src/main/java/ru/marina/tshop/products/FilterProductResponse.java

```diff
@@ -0,0 +1,8 @@
+package ru.marina.tshop.products;
+
+import java.util.List;
+
+class FilterProductResponse {
+    public List<Product> products;
```

Comment:
There has to be either private fields with getters / setters or you could use Lombok.